toolchain/lower/function_context.cpp

```diff
@@ -60,11 +60,16 @@ auto FunctionContext::LowerInst(SemIR::InstId inst_id) -> void {
   CARBON_VLOG() << "Lowering " << inst_id << ": " << inst << "\n";
   builder_.getInserter().SetCurrentInstId(inst_id);
   switch (inst.kind()) {
+#define CARBON_SEM_IR_INST_KIND_CONSTANT_ALWAYS(Name)
 #define CARBON_SEM_IR_INST_KIND(Name)                     \
   case SemIR::Name::Kind:                                 \
     Handle##Name(*this, inst_id, inst.As<SemIR::Name>()); \
     break;
 #include "toolchain/sem_ir/inst_kind.def"
+
+    default:
+      CARBON_FATAL() << "Missing constant value for constant instruction "
```

Comment:
Is this missing, or not expected to be called with an always-constant instruction?

Reply:
`LowerBlock` would skip an always-constant instruction. But I think this'd be clearer if I moved that `if` into `LowerInst`; I've done that.